internal/manifests/manifests.go

```diff
@@ -353,6 +353,12 @@ func (m *Manifests) validateAllowedToModifyManifests(ctx context.Context, cluste
 }
 
 func (m *Manifests) validateUserSuppliedManifest(ctx context.Context, clusterID strfmt.UUID, manifestContent []byte, fileName string) error {
+	// Ultimately manifests are consumed into a ConfigMap. The max permitted size of a ConfigMap is 1MiB
```

Comment:
config map is not exactly a correct term. `MachineConfig` for example is a manifest as well but it's not a configmap, i would describe it as a etcd resource in k8s is limited to 1MB

Reply:
Sorry, I meant to say "stored in a configmap", have updated the code

Reply:
OK, changed this now to say
`// etcd resources in k8s are limited to 1MiB`

Reply:
What is your source that etcd resources are limited to 1MiB? All I see is docs talking about configmaps being limited

Reply:
>Sorry, I meant to say "stored in a configmap", have updated the code

Stored in what configmap?

Reply:
What is your source that etcd resources are limited to 1MiB? All I see is docs talking about configmaps being limited

Reply:
My source is this internal conversation https://redhat-internal.slack.com/archives/CUPJTHQ5P/p1686135268686879
Where it is acknowledged that the etcd max size is 1.5MiB
https://etcd.io/docs/v3.5/dev-guide/limit/#request-size-limit
However, also mentions that the effective limit is 1MiB for ConfigMaps
https://kubernetes.io/docs/concepts/configuration/configmap

Is there a massive concern to setting the manifest limit to 1MiB ? Do you want to make the distinction between different types of content and change the limit accordingly?

Should we try to find out more about what the limit should be, who should we ask?

Reply:
If we set the limit to 1MiB then it would be possible to upload a manifest content of this size and that might lead to an attempt to create a config map of this size.

That's why I think we should lower it to 1MiB as this is the lowest value in common between ConfigMaps and etcd resources.

If we want to analyse manifest content to determine what size should be uploaded, I think this would get complicated quickly.

So I think it's best to set to 1 MiB

Reply:
>Is there a massive concern to setting the manifest limit to 1MiB 

No, I just want to understand why we are doing what we're doing

>Do you want to make the distinction between different types of content and change the limit accordingly?

Not necessarily, but if we don't do that, we need to at-least acknowledge in a comment explicitly that our blanket limit of 1MiB is too conservative but is done to easily have the correct limit also configmaps without worrying about the exact type of resource

Reply:
I have updated the comments and commit message to reflect this concern